libs/patterns.py

```diff
@@ -76,13 +126,13 @@ def __init__(
         # self._logger.setLevel(logging.DEBUG)
         try:
             patterns = patterns_path.read_text()
-        except Exception as e:
-            raise Exception(f'Unable to read patterns from file {patterns_path.absolute()}: {e}')
+        except Exception as err_msg:
+            raise Exception(f'Unable to read patterns from file {patterns_path.absolute()}: {err_msg}')
         else:
             try:
-                self._patterns = json.loads(patterns)
-            except Exception as e:
-                raise Exception(f'Unable to load json data: {e}')
+                self._patterns = json.loads(yaml_2_json(patterns))
```

Comment:
Why do you need to convert yaml to json ? Isn't  yaml good enough to generate a nice dictionnary?

Reply:
It's a migration in two parts :( Maybe it's easy but I didn't wanted to change everything

Reply:
Okay, thank you!